app/apollo/resolvers/channel.js

```diff
@@ -68,7 +68,7 @@ const channelResolvers = {
       const queryFields = GraphqlFields(fullQuery);
       const { models, me, req_id, logger } = context;
       const queryName = 'channelByName';
-      logger.debug({req_id, user: whoIs(me), orgId, name}, `${queryName} enter`);
```

Comment:
Searching the logs is still possible and arguably better with the searchable values put in separate log json fields -- instead of raw text search, one can search for `queryName:channels` and get all the log messages from that function.  Putting values to be searched in the message text is not a good practice -- the message should be static when possible, even translatable ideally, and the 'data' associated with the log should be in separate json.  It is inline with the intent, and is an improvement on the original.

Reply:
"queryName:channels" will not get all messages from that function unless all messages have the field. I feel that unnecessary changes based on an opinion that is not inline with the rest of the codebase just increase the scope of PRs without adding any value.

Reply:
removed